appeals/web/src/server/app/config/session.js

```diff
@@ -1,10 +1,68 @@
-import config from '@pins/appeals.web/environment/config.js';
+import config from '@pins/applications.web/environment/config.js';
```

Comment:
should be from `appeals`?

Reply:
sorry just noticed this is running from `apps/api`... Should we duplicate or maybe move to packages if possible?

Reply:
Good catch!